shiksha-api/app-service/tests/unit/services/test_general_chat_service.py

```
@@ -252,7 +252,7 @@ async def test_call_formats_messages_correctly(
                 ConversationMessage(role=MessageRole.USER, message="How are you?")
             ]
 
-            async for _ in service(messages): pass
+            async for _ in service(messages, user_id="test-user-1"): pass
```

Ruff (0.15.18)

[error] Multiple statements on one line (colon)

(E701)